frontend/src/components/c-summary-charts.vue

```diff
@@ -139,6 +139,18 @@
           v-if="sortGroupSelection.includes('totalCommits')"
         ) {{ getPercentile(i) }} %&nbsp
         span.tooltip-text.right-aligned {{ getPercentileExplanation(i) }}
+      .summary-charts__title--tags(
+        v-if="filterGroupSelection === 'groupByRepos' && viewRepoTags"
+      )
```

Comment:
Can we store this boolean in a computed variable with a name that describes what this condition is? (similar to `isChartGroupWidgetMode` and `isChartWidgetMode` in this file)

Reply:
fixed!